docker-compose.test.yaml

```diff
@@ -68,3 +68,25 @@ services:
     environment:
       - POSTGRES_USER=root
       - POSTGRES_PASSWORD=root
+
+  test-db-for-obsolete-moodle:
```

Comment:
I wanted to avoid setting up additional container just for the test service purpose, but setting up sqlite with old apach/php/moodle posed greater problem that I at first thought.
I decided it's not worth investing more time into one container setup, so moved to this solution.